src/lib.rs

```diff
@@ -831,10 +831,11 @@ mod css_color_tests {
         );
     }
 
+    #[macro_use]
     mod conversions {
         use crate::{Angle, Ratio, HSL, HSLA, RGB, RGBA};
 
-        trait ApproximatelyEq {
+        pub trait ApproximatelyEq {
```

Comment:
I am unsure if this is a bad thing to do/not a best practice. Am thinking that I could create a `operations` macro for color operation tests at some point, but didn't want to do that refactor until it really seemed necessary (and until all the operations are done). 

Just doing this now so that I have access to `assert_approximately_eq!()` in other color operation tests, but would like to know what is the right thing to do here. cc @chancancode